test/Android/androidtest.cmd

```diff
@@ -123,10 +123,19 @@ if "%_Verbose%"=="1" (
 if "%_TestsToRun%"=="" goto RunAll
 goto RunArgs
 
+:FindDeviceId
+pushd %_SdkRoot%
+for /f "tokens=1,6" %%i in ('platform-tools\adb devices -l ^| find /i "VS Emulator"') do (
+	echo Run on device: %%i %%j
+    set _DeviceId=%%i
+	popd
```

Comment:
Would it make sense to only set the Platform to x86 if you are using this for loop?